clusterloader2/pkg/state/namespaces.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Since this is only one file now, maybe we can name it as `state.go`?

Reply:
renamed to `stage.go`.